src/main/java/org/isf/menu/manager/UserBrowsingManager.java

```diff
@@ -151,26 +144,23 @@ public void deleteUser(User user) throws OHServiceException {
 	// operation was too heavy handed.
 	/**
 	 * Increase the number of failed login attempts for {@link User}.
-	 *
 	 * @param user the {@link User}
 	 */
-	public void increaseFailedAttempts(User user) {
+	public void increaseFailedAttempts(User user) throws OHServiceException {
 		int newFailAttempts = user.getFailedAttempts() + 1;
 		ioOperations.updateFailedAttempts(user.getUserName(), newFailAttempts);
 	}
 
 	/**
 	 * Reset the number of failed login attempts to zero for {@link User}.
-	 *
 	 * @param user the {@link User}
 	 */
-	public void resetFailedAttempts(User user) {
+	public void resetFailedAttempts(User user) throws OHServiceException {
```

Comment:
This is impacting GUI

Reply:
I'm going to revert this one ! We don't need it !